counter-app/src/main/java/camp/nextstep/edu/counter/CounterViewModel.kt

```diff
@@ -0,0 +1,37 @@
+package camp.nextstep.edu.counter
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class CounterViewModel : ViewModel() {
+
+	private val _count: MutableLiveData<Int> = MutableLiveData(0)
+	val count: LiveData<Int>
+		get() = _count
+
+
+	private val _countDownFailure: MutableLiveData<Boolean> = SingleLiveEvent(false)
+	val countDownFailure: LiveData<Boolean>
+		get() = _countDownFailure
+
+	fun countUp() {
+		val countSnapshot = _count.value ?: return
+
+		_count.value = countSnapshot + 1
+	}
+
+	fun countDown() {
+		val countSnapshot = _count.value ?: return
+
+		if (countSnapshot <= 0) {
+			_countDownFailure.value = true
```

Comment:
```suggestion
			_countDownFailure.value = Unit
```
Nit: _countDownFailure가 false를 보내는 경우는 없습니다.
이런경우 boolean을 보내는 것이 큰 의미를 가지지 않을 수 있어요.
해당 라이브데이터에 값이 변화되었다는 것만 알면 되기 때문에 LiveData가 수신하는 타입을 Unit으로 주어서 별도의 boolean을 체크하지 않아도 괜찮습니다.

Reply:
아하..! Unit 으로 이벤트를 던져줄 수 도 있겠네요 ㅎㅎㅎ 감사합니다!!